test/document_stores/test_document_store.py

```diff
@@ -389,14 +389,14 @@ def test_get_documents_by_id(document_store: BaseDocumentStore):
 
 def test_get_document_count(document_store: BaseDocumentStore):
     documents = [
-        {"content": "text1", "id": "1", "meta_field_for_count": "a"},
+        {"content": "text1", "id": "1", "meta_field_for_count": "c"},
         {"content": "text2", "id": "2", "meta_field_for_count": "b"},
         {"content": "text3", "id": "3", "meta_field_for_count": "b"},
         {"content": "text4", "id": "4", "meta_field_for_count": "b"},
     ]
     document_store.write_documents(documents)
     assert document_store.get_document_count() == 4
-    assert document_store.get_document_count(filters={"meta_field_for_count": ["a"]}) == 1
+    assert document_store.get_document_count(filters={"meta_field_for_count": ["c"]}) == 1
```

Comment:
Yeah, that was an odd one, wasn't it? :-)
It took us a while to see why that test failing with "a". It turns out, that "a" is a stopword in English in Weaviate, so a filter with a single stopword is no filter at all and such query with that filter fails.
So I have changed it to filter by "c" instead of "a", as "c" is NOT a stopword.

See conversation at the Weaviate Slack channel: https://weaviate.slack.com/archives/C017EG2SL3H/p1658255777389299

Reply:
Oh boy 🙈 
Do you think we should change those strings to a more meaningful example? Or leave a comment for posterity warning against using `a`?

Reply:
Yes, I was thinking that maybe I should go through all the tests and change the "a" documents to something more meaningful, but then I have realized that that likely shouldn't be part of this PR, but a separate one.
So yes, it might be worth having a separate refactoring PR - where in the notes of the PR the reasoning behind the refactoring of the tests could be properly explained.